src/main/java/tictactoe/GameOutcome.java

```diff
@@ -0,0 +1,11 @@
+package tictactoe;
```

Comment:
Does this enum need a `type` field? Why not:

```
public enum GameOutcome {
  TIE;
}
```

Reply:
I don't think it does, as `TIE` would be the same as the type